src/raft_log.rs

```diff
@@ -483,26 +490,23 @@ impl<T: Storage> RaftLog<T> {
         }
 
         if low < self.unstable.offset {
-            let stored_entries =
-                self.store
-                    .entries(low, cmp::min(high, self.unstable.offset), max_size);
-            if stored_entries.is_err() {
-                let e = stored_entries.unwrap_err();
-                match e {
+            let unstable_high = cmp::min(high, self.unstable.offset);
+            match self.store.entries(low, unstable_high, max_size) {
+                Err(e) => match e {
                     Error::Store(StorageError::Compacted) => return Err(e),
                     Error::Store(StorageError::Unavailable) => panic!(
                         "{} entries[{}:{}] is unavailable from storage",
-                        self.tag,
-                        low,
-                        cmp::min(high, self.unstable.offset)
+                        self.tag, low, unstable_high,
                     ),
                     _ => panic!("{} unexpected error: {:?}", self.tag, e),
+                },
```

Comment:
Can you send a separate PR with such identical changes? I think they can be merged more quickly and make this PR more focus.